README.md

```diff
@@ -31,15 +33,15 @@ Here you find more detailed information on:
 
 Clone this repository:
 
-        git clone ssh://git@stash.desy.de:7999/gul/libgul.git
-        cd libgul
+        git clone https://github.com/gul-cpp/gul14.git
+        cd gul14
 
 Use meson to configure the build you want to have. A list of the
 [build options](#Build-configuration-switches) is given below. Build directory name can be
 what you want. If you use directories starting with 'build' they will be ignored by git.
 Examples:
 
-        meson build
+        meson builddir
```

Comment:
Maybe we should use `<builddir>` instead, since one can also use e.g. `/tmp/build` which is on modern linux distros in memory.

Reply:
But since this is an example, people probably expect to be able to copy&paste it directly, no? It is not a command synopsis or anything like that.

Reply:
But why `builddir`? In the next lines it is always varying...
* `build_without_tests`
* `build_rel`
* `build_rel_unity`

Reply:
Okay, back to `build` it is.